src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseApi.java

```diff
@@ -253,11 +305,11 @@ private CCDCase convertToCCDCase(Map<String, Object> mapData) {
         return jsonMapper.fromJson(json, CCDCase.class);
     }
 
-    private Claim mapToClaim(Map.Entry<Long, Map<String, Object>> entry) {
-        Map<String, Object> entryValue = entry.getValue();
```

Comment:
this was too fancy before converting it to a map meant we lost the order that came back from CCD, this is why our cases were appearing in a random order on the dashboard